web-wallet/svelte.config.js

```diff
@@ -2,11 +2,16 @@ import adapter from "@sveltejs/adapter-static";
 import autoprefixer from "autoprefixer";
 import postCSSNested from "postcss-nested";
 import preprocess from "svelte-preprocess";
+import { loadEnv } from "vite";
+
+const env = loadEnv("", process.cwd());
+const fallBackBase = env.VITE_BASE_PATH ? `${env.VITE_BASE_PATH}/` : "";
 
 /** @type {import("@sveltejs/kit").Config} */
 const config = {
 	kit: {
-		adapter: adapter({ fallback: "index.html" })
+		adapter: adapter({ fallback: `${fallBackBase}index.html` }),
+		paths: { base: env.VITE_BASE_PATH }
```

Comment:
Shouldn't this also use the `fallBackBase`?

Reply:
> Shouldn't this also use the `fallBackBase` or will just it be `undefined` without affecting the functionality?

1. The `fallBackBase` has an additional trailing slash
2. it shouldn't be `undefined`, but it won't affect the functionality if it is **and** the wallet is served from a root path